docs/source/format/Columnar.rst

```diff
@@ -1019,7 +1024,13 @@ Schematically we have: ::
 In the file format, there is no requirement that dictionary keys
 should be defined in a ``DictionaryBatch`` before they are used in a
 ``RecordBatch``, as long as the keys are defined somewhere in the
-file.
+file. Further more, it is invalid to have more then one non-delta
+dictionary batch per dictionary ID.  Delta dictionaries are applied
+in the order they appear in the file footer.
+
+.. note:: Implementations should check to ensure the dictionary constraints
+  are satisfied.  In future revisions of the specification this requirement
+  might be relaxed.
```

Comment:
I don't follow what this note is getting at

Reply:
I think I should simply remove this. I included it because there was some discussion on if dictionary replacement should be allowed in the file format